### ISSUE TITLE
Negative CE as regularizer in warmup iters

### ISSUE THREAD

**Issue by mohitsharma29** [xyupeng/BETA#2]
Hey

The paper mentions that L_ne (Equation 2) is used as a regularizer in the warmup phase. However, I could not find that in the warmup function in the train_BETA file. Is it the case that you just used the cross entropy loss for warmup training?
